addons/stock_account/models/stock_move_line.py

```diff
@@ -72,3 +72,26 @@ def _create_correction_svl(self, move, diff):
             stock_valuation_layers |= move._create_dropshipped_returned_svl(forced_quantity=abs(diff))
 
         stock_valuation_layers._validate_accounting_entries()
+
+    @api.model
+    def _should_exclude_for_valuation(self):
+        """
+        Determines if this move line should be excluded from valuation based on its ownership.
+        :return: True if the move line's owner is different from the company's partner (indicating
+                it should be excluded from valuation), False otherwise.
+        """
+        self.ensure_one()
+        if self.owner_id and self.owner_id != self.company_id.partner_id:
```

Comment:
In general, this structure:

```
if condition:
    return True
return False
```

can be replaced with:

```
return condition
```

That goes for the other 2 methods as well ;)

Reply:
@pfertyk Actually, I considered your suggestion in the first place. But for design consistency, I decided to use currency approach. This is how odoo standard use for this kind of function.
https://github.com/odoo/odoo/blob/ed890131f5bff8264e152c77ae5d3757f2dd7bfb/addons/stock_account/models/stock_location.py#L25-L32

Reply:
@AungKoKoLin1997 There are far more occurrences of what @pfertyk suggested in the code. Let's avoid bloating the code for nothing. :slightly_smiling_face: